api/v1/slurmcluster_types.go

```diff
@@ -73,6 +73,36 @@ type SlurmClusterSpec struct {
 	// https://slurm.schedmd.com/slurm.conf.html#SECTION_PARTITION-CONFIGURATION
 	// +kubebuilder:validation:Optional
 	PartitionConfiguration PartitionConfiguration `json:"partitionConfiguration,omitempty"`
+
+	// SlurmConfig represents the Slurm configuration in slurm.conf. Not all options are supported.
+	//
+	// +kubebuilder:validation:Optional
+	SlurmConfig SlurmConfig `json:"slurmConfig,omitempty"`
+}
+
+// SlurmConfig represents the Slurm configuration in slurm.conf
+type SlurmConfig struct {
```

Comment:
Can we add the `TaskPluginParam` with the default value `OOMKillStep,Verbose`?

Reply:
If we want to kill by OOM an entire step set memory.oom.group = 1 to step for OOMKillStep